index.html

```diff
@@ -5,41 +5,64 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style1.css">
     <head>
         <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css" integrity="sha384-*********************" crossorigin="anonymous">
       </head>
       
     <title>Portfolio</title>
 </head>
 <body>
+<div class="font-div">
     <header>
         <a class="logo">MZ</a>
         <nav>
         <i id="menu"  class="fas fa-bars"></i> 
         </nav>
   </header>
-  <section >
-        <div class="item">
-            <h1> I'm Massa Zoumanigui <br> Glad to see you</h1>
-        </div>
-    <div class="item">
-        <p>I’m a software developer! I can help you build a product , feature or website Look through <br> 
-            some of my work and experience! If you like what you see and have a project you need <br>
-            coded, don’t hestiate to contact me</p>
-    </div>
+
+
+
+
+
+<div class="finish">
```

Comment:
- [ ] [**OPTIONAL**] Kindly keep doing the  great work by remove this bank lines 👍   it  is crucial to clean readable code